wdl-analysis/src/handlers/completions.rs

```diff
@@ -698,7 +758,38 @@ fn add_struct_completions(document: &Document, items: &mut Vec<CompletionItem>)
             detail: Some(format!("struct {name}")),
             documentation: provide_struct_documentation(s, &root).and_then(make_md_docs),
             ..Default::default()
-        })
+        });
+
+        if let Some(ty) = s.ty()
+            && let Some(struct_ty) = ty.as_struct()
+        {
+            let members = struct_ty.members();
+            if !members.is_empty() {
```

Comment:
If you would, pull this out into its own method alongside `build_call_snippet` (maybe `build_struct_snippet`).